hat/assets/js/apps/Iaso/domains/orgUnits/details.tsx

```diff
@@ -142,13 +154,11 @@ const OrgUnitDetail = () => {
             return formatMessage(MESSAGES.newOrgUnit);
         }
         if (currentOrgUnit?.org_unit_type_name) {
-            return (
-                `${currentOrgUnit?.name}${
-                    currentOrgUnit.org_unit_type_name
-                        ? ` - ${currentOrgUnit.org_unit_type_name}`
-                        : ''
-                }` ?? ''
-            );
+            return `${currentOrgUnit?.name}${
```

Comment:
if `currentOrgUnit` is `null` this will show `"null"`
```suggestion
            return `${currentOrgUnit?.name??''}${
```

Reply:
this one @beygorghor